src/ui/common/src/components/pages/components/SaveDetails.tsx

```diff
@@ -0,0 +1,93 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import { isGoogleSheetsLoadParams, isRelationalDBLoadParams, isS3LoadParams, LoadParameters } from '../../../utils/operators';
+
+type SaveDetailsProps = {
+  parameters: LoadParameters;
+};
+
+export const SaveDetails: React.FC<SaveDetailsProps> = ({
+  parameters
+}) => {
+  let paramsDisplay = null;
+  if (parameters) {
+    if (isRelationalDBLoadParams(parameters)) {
```

Comment:
Can you make this into a switch statement please?

Reply:
Since I am passing it into a different function in each if statement (isRelationalDBLoadParams, isGoogleSheetsLoadParams, etc.) and it isn't an equality check, I'm not too sure what would be the best way to go about that. Would appreciate your thoughts!

Reply:
Okay the changes I will make are:
1) create a helper function that will give me the specific load type as a string so I can do a switch statement
2) make each of them its own components